### PR TITLE
Service Request GitHub template

### PR DESCRIPTION
**Pull request by bhgrant8**
Adds a basic service request/issue template for users to report issues experienced with our infrastructure or backend services

### CODE REVIEW

**Review by jaronheard — APPROVED** [review]
@BrianHGrant This looks good. I recommend we merge and iterate

**Comment by bhgrant8** [comment]
@MikeTheCanuck going to ask for your input here, as main owner/maintainer of this repo whether you feel appropriate move to basic triage backend and service issues through this repo, before moving towards other projects? 

This option came out of discussion with @DingoEatingFuzz

File: .github/ISSUE_TEMPLATE/service_request_template.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+**Description of issue**
+
+> Describe the issue or need you are experiencing
+
+**Error Message/Logs**
+
+> Can you provide the error message or other logs which indicate the error. Examples: Browser's Javascript Console, AWS CloudWatch logs, terminal output. Please provide in code blocks
+
+**Recreation Steps**
```
Comment (MikeTheCanuck) [inline .github/ISSUE_TEMPLATE/service_request_template.md]: In my universe the overwhelmingly more common phrase is "Reproduction Steps" - I recommend we not confuse those who are familiar with the repro terminology

**Comment by MikeTheCanuck** [comment]
Beyond one phrasing recommendation this is *great* - far more comprehensive and actionable than many "support request" or "problem report" templates I've seen used in the industry.  Looking forward to being able to resolve real problems based on this approach.

**Comment by bhgrant8** [comment]
I made update, going ahead with merge

**Pull request merged**